### PR TITLE
NOC-27 Print styling - Impact of Guidance

### PR DESCRIPTION
**Pull request by wa-rren-dev**
https://nicedigital.atlassian.net/browse/NOC-27

- Rename SASS references

### CODE REVIEW

File: src/components/_footer.scss [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 /// Footer component
 .footer {
```
Comment (ediblecode) [inline src/components/_footer.scss]: We can remove the footer now can't we? Now that we've rolled out the global nav?

File: src/components/_hero.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Extension to the Design System hero component
 .hero {
   // sass-lint:disable no-duplicate-properties
-  margin-bottom: rem($spacing-large);
+  margin-bottom: rem($nds-spacing-large);
 
   // Hack to 'break out' of the container (layout) for hero components
```
Comment (ediblecode) [inline src/components/_hero.scss]: Didn't yout put the 'break out' CSS directly in the design system in v1, in which case we can remove this break out?

File: src/components/_hero.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // Extension to the Design System hero component
 .hero {
   // sass-lint:disable no-duplicate-properties
-  margin-bottom: rem($spacing-large);
+  margin-bottom: rem($nds-spacing-large);
```
Comment (ediblecode) [inline src/components/_hero.scss]: Maybe we should just change https://github.com/nice-digital/nice-design-system/blob/v1/components/nds-hero/scss/hero.scss#L13 to be a large by default then there's no need to extend/ovewrite it here?

File: src/components/_icons.scss [hunk 4]
```diff
@@ -6,94 +6,62 @@
...
-  &.icon-android:before {
-    content: nice-icon(android);
+  &.icon-android {
+    @include nice-icon(android);
   }
 
-  &.icon-apple:before {
-    content: nice-icon(apple);
+  &.icon-apple {
+    @include nice-icon(apple);
```
Comment (ediblecode) [inline src/components/_icons.scss]: This bumps up the size of the output CSS quite a lot, byt changing from content to mixin. Probably not a big deal after gzipping but wonder if using the content approach is more performant?

Comment (ediblecode) [reply src/components/_icons.scss]: The base mixin is now `nice-icons-base` if you need it

File: src/components/_prettyprint.scss [hunk 5]
```diff
@@ -2,7 +2,7 @@
 // sass-lint:disable no-combinators
 .prettyprint .tag {
   background: inherit;
-  font-size: inherit;
+  nds-font-size: inherit;
```
Comment (ediblecode) [inline src/components/_prettyprint.scss]: This should be font-size

File: src/components/_tophat.scss [hunk 6]
```diff
@@ -11,15 +11,15 @@ body {
 
       .nice-search {
         float: none;
-        margin: em($spacing-medium) auto;
+        margin: em($nds-spacing-medium) auto;
```
Comment (ediblecode) [inline src/components/_tophat.scss]: I think we can get rid of this whole file now we're using global nav

File: src/components/_typography.scss [hunk 7]
```diff
@@ -14,7 +14,7 @@ abbr {
   }
 
   &.initialism {
-    font-size: 90%;
+    nds-font-size: 90%;
```
Comment (ediblecode) [inline src/components/_typography.scss]: Should be font-size

File: src/components/_visibility.scss [hunk 8]
```diff
@@ -86,7 +86,7 @@
   background-color: transparent;
   border: 0;
   color: transparent;
-  font-size: 0;
+  nds-font-size: 0;
```
Comment (ediblecode) [inline src/components/_visibility.scss]: Should be font-size

File: src/nice.bootstrap/_breadcrumbs.scss [hunk 9]
```diff
@@ -26,8 +26,8 @@
   .divider {
     color: #004650;
     font-family: font-family(sans);
-    font-size: 80%;
-    margin: em(0 $spacing-x-small);
+    nds-font-size: 80%;
```
Comment (ediblecode) [inline src/nice.bootstrap/_breadcrumbs.scss]: Should be font-size

File: src/nice.bootstrap/_button-group.scss [hunk 10]
```diff
@@ -2,7 +2,7 @@
 // See http://nhsevidence.github.io/NICE.Bootstrap/Guide.FormElements.html#buttonGroups
 .btn-group {
   display: inline-block;
-  font-size: 0;
+  nds-font-size: 0;
```
Comment (ediblecode) [inline src/nice.bootstrap/_button-group.scss]: Should be font-size

File: src/nice.bootstrap/_button-group.scss [hunk 11]
```diff
@@ -31,11 +31,11 @@
 }
 
 .btn-toolbar {
-  font-size: 0;
-  margin: rem($spacing-medium);
+  nds-font-size: 0;
```
Comment (ediblecode) [inline src/nice.bootstrap/_button-group.scss]: Should be font-size

File: .grunt-tasks/sass.js [hunk 12]
```diff
@@ -1,8 +1,11 @@
+var tilde_importer = require("grunt-sass-tilde-importer");
```
[the remark this answers is not in the document]
Comment (greypiglet) [reply .grunt-tasks/sass.js]: Have added some emphasis to the existing copy and included a link to the grunt plugin

**Comment by greypiglet** [comment]
https://nicedigital.atlassian.net/browse/NOC-27

File: src/nice.bootstrap/_lists.scss [hunk 13]
```diff
@@ -62,15 +62,15 @@ ul.alt {
...
-    margin-left: rem($spacing-medium);
-    min-height: rem($spacing-small);
+    margin-left: rem($nds-spacing-medium);
+    min-height: rem($nds-spacing-small);
     position: relative;
 
     &:before {
-      color: $colour-icons;
+      color: $nds-colour-icons;
       content: get-glyph(bullet);
```
Comment (ediblecode) [inline src/nice.bootstrap/_lists.scss]: Shouldn't this be `nds-get-glyph`?

File: .gitignore [hunk 14]
```diff
@@ -47,3 +47,4 @@ source-css
 
 
 dist
+.vscode/settings.json
```
Comment (ediblecode) [inline .gitignore]: Please don't ignore settings!

**Pull request closed without merging**